src/Service/CaptureServiceImpl.cpp

```diff
@@ -227,8 +228,10 @@ static ProducerCaptureEvent CreateCaptureStartedEvent(const CaptureOptions& capt
     ERROR("%s", executable_path_or_error.error().message());
   }
 
-  capture_started->mutable_capture_options()->CopyFrom(capture_options);
   capture_started->set_capture_start_timestamp_ns(capture_start_timestamp_ns);
+  auto version = orbit_version::GetVersionNumber();
```

Comment:
nit. auto -> orbit_version::Version? For simple types I really prefer not using auto. go/cstyle#Type_deduction

Reply:
done.